packages/cadl-ranch-specs/http/type/enum/extensible/main.tsp

```diff
@@ -7,27 +7,29 @@ using TypeSpec.Http;
 namespace Type.Enum.Extensible;
 
 @doc("Days of the week")
-enum DaysOfWeekExtensibleEnum {
+union DaysOfWeekExtensibleEnum {
```

Comment:
`getKnownValue(): DaysOfWeekExtensibleEnum;` likely have problem even in TCGC.

tsp compiler would think this be multiple responses. (`HttpOperationResponse` which contains multiple `HttpOperationResponseContent` in `responses` property)

Reply:
Left a message in https://github.com/microsoft/typespec/issues/2848#issuecomment-1952088905

It is pretty hard for emitter to handle this case, when the Union is already taken apart to Type.

Reply:
Changed to use explicit `@body`
```ts
  getKnownValue(): {
    @body body: DaysOfWeekExtensibleEnum;
  };
```
on Timothee's advice. This should unblock emitter.